rmgpy/data/solvation.py

```diff
@@ -2011,13 +2007,15 @@ def get_Kfactor_parameters(self, delG298, delH298, delS298, solvent_name, T_tran
         The conversion between dGsolv estimate from the Abraham and K-factor is shown below:
                 dGsolv = RTln(K-factor * rho_g / rho_l)
         where rho_g is the saturated gas phase density of the solvent.
+        See the RMG documentation "Liquid Phase Systems" section on a temperature-dependent model for more details.
 
         Args:
             delG298 (float): solvation free energy at 298 K in J/mol.
             delH298 (float): solvation enthalpy at 298 K in J/mol.
             delS298 (float): solvation entropy at 298 K in J/mol/K.
             solvent_name (str): name of the solvent that is used in CoolProp.
-            T_trans_factor (float): a temperature
+            T_trans_factor (float): a temperature [K] for transitioning from the first piewise function to the
```

Comment:
Typo in Piecewise

Reply:
Thanks! I will fix this typo and then squash the fixup

Reply:
I just fixed the typo, squashed the fixup, and then force-pushed.